pywbem/_subscription_manager.py

```diff
@@ -837,6 +841,18 @@ def add_filter(self, server_id, source_namespace, query,
             filter instance, or `None` if the `filter_id` parameter is
             specified.
 
+          source_namespace (:term:`string):
+            Optional source namespace of the indication filter. If the
+            parameter is provided the value will be inserted into the
+            CIM_IndicationFilter SourceNamespace property. Otherwise the
+            SourceNamespace property will not be created. The SourceNamespace
+            property is deprecated in the DMTF CIM_IndicationFilter class in
+            favor of SourceNamespaces( see the SourceNamespaces parameter
```

Comment:
Use the function parameter name: " (see the source_namespaces parameter above)"

Reply:
DONE